ts/src/application.ts

```diff
@@ -26,30 +27,34 @@ class TodoSequence extends Sequence {
     try {
       // 1. Find the endpoint/route
       // Throws HttpErrors.NotFound when there is no route matching the request method and path
-      const {controllerName, methodName, spec, pathParams} = findRoute(request);
+      const {controllerName, methodName, spec, pathParams} = this.findRoute(request);
       
-      // 2. Authenticate & authorize
+      // 2. Authenticate and bind current user
       const user = await this.authenticate(request);
-      
-      // 3. Authorize the operation
-      // TODO(bajtos) how to pass target model id for owner-based ACLs?
-      await this.authorize(controllerName, methodName, user);
+
+      // Anything produced / determined during the sequence
+      // should be bound into the request context
+      this.context.bind('authentication.user').to(user);
```

Comment:
As I have envisioned the sequence steps, and also how the current implementation of `findRoute` works, these bindings should be created by the sequence steps. See https://github.com/strongloop/loopback-next/blob/67841d32131b896e8b09cb0b4e171f8ca067bab3/packages/core/src/http-handler.ts#L54-L72

Not a big deal, your proposal is fine too, and we can tune this incrementally.

From what I have observed while working on the initial Sequence implementation, there are certain implementation constraints than have to be accounted for in the design of the acceptance tests. I find it difficult to foresee them in advance, before there is at least a partial implementation to back the design. In that light, I am proposing to move the acceptance tests back into loopback-next repository and include changes in the implementation when changing acceptance tests.